lib/erlef/wg.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Erlef.WG do
+  defstruct name: "", slug: "", description: "", email: "", formed: nil
+
```

Comment:
Do you think `email` should default to `nil` so we can check for that in the view vs `String.length/1`?

Reply:
yes, good call.